DEVELOPMENT.md

```diff
@@ -0,0 +1,70 @@
+# Uploadcare Widget development guide
+
+Clone the repository.
+
+### Enviroment
+
+You need a working Ruby 2.0.0 and above environment with [Bundler](http://bundler.io).
+
+#### Vagrant
+
+If you don't want to raise the environment on your machine, you can use [Vagrant](https://www.vagrantup.com/).
+Just [install Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/) or [other provider](https://www.vagrantup.com/docs/getting-started/providers.html).
+
+After open command line, go to this folder and run:
+
+    vagrant up
+    vagrant ssh
+
+#### Local environment (without Vagrant)
+
+Inside folder run
+
+    bundle install
+    cd ./test/dummy
+    bundle install
+
+### Build
+
+* `bundle exec rake js:latest:build` to build assets
+  to the **pkg/latest** directory (with the “latest” suffix).
+* `bundle exec rake js:release:build` to build assets
+  to the **pkg/version** directory (with the current version suffix).
+  The version is specified in `lib/uploadcare-widget/version.rb`.
+
+### Development
+
+Go to `test/dummy/`. There is a simple Rails app. Run it:
+
+    bundle exec rails server
+
+Open http://127.0.0.1:3000/. Follow any link.
+There's going to be a widget or three. Edit code and reload the page :-)
```

Comment:
wut? the phrase goes like "a widget or two."
"Edit code and..." >> Modify code? Mutate code?